### PR TITLE
Add ECS section in Troubleshoot page

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Document ECS troubleshooting tips](https://trello.com/c/GGWuYB0I/2122-document-ecs-troubleshooting-tips)

File: _help/troubleshooting.md [hunk 1]
```diff
@@ -19,3 +19,32 @@ Run `convox apps info` to find the load balancer endpoints for your application.
...
+
+## Still having trouble?
+
+Some good places to search are:
+
+- this site, via [google](https://www.google.com/search?q=site%3Aconvox.com) or the search box above
+- the history of our [public Slack channel](https://invite.convox.com/)
+- [GitHub issues](https://github.com/convox/rack/issues)
+
+If you still need help, free to:
```
Comment (ianmalott) [inline _help/troubleshooting.md]: I think you a word.

File: _help/troubleshooting.md [hunk 2]
```diff
@@ -19,3 +19,32 @@ Run `convox apps info` to find the load balancer endpoints for your application.
...
+
+## ECS
+
+If you're encountering problems like any of the following:
+
+- deployments seem stuck in `UPDATE_IN_PROGRESS` CloudFormation state
+- your Rack seems stuck in `converging`
+- your deploys seem stuck with services continuously being killed via SIGKILL and restarted
+
+... then you should check the ECS section of your AWS console. To do so, log into the AWS ECS web UI and click the Cluster for your Rack. You'll be taken to a screen with a list of **Services**. Click on the name of the `ServiceWeb` service; you'll be taken to a screen with an **Events** tab. The events there should provide some clues about why `ServiceWeb` isn't stabilizing.
```
Comment (ianmalott) [inline _help/troubleshooting.md]: Might be nice to link  [AWS ECS web UI] to https://console.aws.amazon.com/ecs/home for convenience.

Comment (soulshake) [inline _help/troubleshooting.md]: Pushed an update to address this, thanks.

File: _help/troubleshooting.md (hunk 1 above)
Comment (soulshake) [inline _help/troubleshooting.md]: Fixed thanks

**Pull request merged**